gradle/libs.versions.toml

```diff
@@ -20,7 +20,8 @@ microprofile-rest-client = "3.0"
 jacoco = "0.8.12"
 glassfish-expressly = "5.0.0"
 
-opa-binary = "0.63.0"
+# please keep this OPA binary version in sync with the one used in the ZAC Helm chart (values.yaml file)
+opa-binary = "0.69.0"
```

Comment:
this conflicts with Hrsto's PR - can you please agree on how this should be used

Reply:
Removed  this change as part of conflict resolution